client/client.go

```diff
@@ -86,6 +86,10 @@ type Client interface {
 	// ScatterRegion scatters the specified region. Should use it for a batch of regions,
 	// and the distribution of these regions will be dispersed.
 	ScatterRegion(ctx context.Context, regionID uint64) error
+	// ScatterRegionWithGroup scatters the specified region. Should use it for a batch of regions , and the distribution
+	// of these regions will be dispersed If they have the same group.
+	// If the group is not defined, it will work as ScatterRegion.
+	ScatterRegionWithGroup(ctx context.Context, regionID uint64, group string) error
```

Comment:
Please consider name it `ScatterRegionWithOption`, so that we can avoid adding new APIs when there is a new feature requirement in the future.
```suggestion
	ScatterRegionWithOption(ctx context.Context, regionID uint64, option Option...) error
```

Reply:
good idea.